devito/operator/operator.py

```diff
@@ -641,16 +641,6 @@ def _soname(self):
 
     @cached_property
     def ccode(self):
-        """
-        Generate C code.
```

Comment:
I would keep this

Reply:
https://github.com/devitocodes/devito/pull/2018#discussion_r997811708